quantum/split_common/matrix.c

```diff
@@ -257,7 +257,7 @@ bool matrix_post_scan(void) {
         static uint8_t error_count;
 
         matrix_row_t slave_matrix[ROWS_PER_HAND] = {0};
-        if (!transport_master(slave_matrix)) {
+        if (!transport_master(matrix + thisHand, matrix + thatHand)) {
```

Comment:
```suggestion
        if (!transport_master(matrix + thisHand, slave_matrix)) {
```

Reply:
@drashna's suggestion fixes my previous issue with the slave not reporting keypresses.

Reply:
fixed in base pr: https://github.com/qmk/qmk_firmware/pull/11046